op_builder/all_ops.py

```diff
@@ -0,0 +1,22 @@
+"""
+Copyright 2020 The Microsoft DeepSpeed Team
+"""
+from deepspeed.accelerator import get_accelerator
+
+# TODO: infer this list instead of hard coded
+# List of all available ops
+__op_builders__ = [
+    get_accelerator().create_op_builder("CPUAdamBuilder"),
```

Comment:
Can we convert all these op names to symbolic constants that can be imported by users to avoid typo bugs?

Reply:
How about following form?

```
from op_builder.builder_names import CPUAdamBuilder

get_accelerator().create_op_builder(CPUAdamBuilder)
```

Reply:
That works too.